pan3d/dataset_builder.py

```diff
@@ -63,18 +66,18 @@ def viewer(self):
             # Build UI
             self._layout = SinglePageWithDrawerLayout(self.server)
             with self._layout as layout:
+                client.Style(open(f"{DIR}/ui/custom.css").read())
```

Comment:
You should use pathlib and Path

```python
from pathlib import Path

BASE_DIR = Path(__file__).parent
CSS_FILE = BASE_DIR / "ui" / "custom.css"

client.Style(CSS_FILE.read_text())
```

Reply:
Thanks, changed in [81ab442](https://github.com/Kitware/pan3d/pull/30/commits/81ab442f8daba987d66b386289633fd0e4a6699c)